fcrepo-event-serialization/src/main/java/org/fcrepo/event/serialization/EventSerializer.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.event.serialization;
+
+import static com.hp.hpl.jena.datatypes.xsd.XSDDatatype.XSDdateTime;
+import static com.hp.hpl.jena.rdf.model.ModelFactory.createDefaultModel;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createProperty;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createResource;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createTypedLiteral;
+import static com.hp.hpl.jena.vocabulary.RDF.type;
+import static com.hp.hpl.jena.vocabulary.DCTerms.identifier;
+import static com.hp.hpl.jena.vocabulary.DCTerms.isPartOf;
+import static org.fcrepo.kernel.api.RdfLexicon.PROV_NAMESPACE;
+import static org.fcrepo.kernel.api.observer.OptionalValues.BASE_URL;
+import static org.fcrepo.kernel.api.observer.OptionalValues.USER_AGENT;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.Resource;
+import org.fcrepo.kernel.api.observer.FedoraEvent;
+
+/**
+ * A basic serialization API for Fedora events
+ * @author acoburn
+ */
+public interface EventSerializer {
+
+    /**
+     * Convert an event to an Rdf Model
+     * @param evt the Fedora event
+     * @return an RDF model representing the event
+     */
+    public static Model toModel(final FedoraEvent evt) {
```

Comment:
Presumably the only difference required to support alternate RDF serializations (that are supported by Jena Models) would involve a parameterized version of your existing `TurtleSerializer`. Do you envision the possibility of creating separate classes for each Jena supported serialization in order to effectively change the string literal found here:
https://github.com/fcrepo4/fcrepo4/pull/1053/files/7bd36eddb3bae3e7a50dc672fdbb29a597fc6e27#diff-abb99f53759398650974df4fa035e481R42